archinstall/lib/profile/__init__.py

```diff
@@ -1,9 +0,0 @@
-from .profile_menu import ProfileMenu, select_greeter, select_profile
-from .profiles_handler import profile_handler
-
-__all__ = [
```

Comment:
@correctmost I know you just merged this change but I run into a circular dependency when loading profile configuration from a config file